apps/test/unit/code-studio/pd/form_components_func/FormControllerTest.js

```diff
@@ -166,8 +156,7 @@ describe('FormController', () => {
         it('Does not submit when the last page has errors', () => {
           setupErrored();
 
-          submitButton().simulate('submit');
-          form.update();
```

Comment:
isolate-components seems to synchronously update the component as events happen, which makes it easy to get rendered data